src/main/java/mate/academy/service/impl/MovieServiceImpl.java

```diff
@@ -24,6 +31,18 @@ public Movie get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        return movieDao.getAll();
+    }
+
+    @Override
+    public Boolean update(Movie movie) {
+        Movie currentMovie = get(movie.getId());
+        movieDao.update(movie);
+        return currentMovie.equals(get(movie.getId()));
```

Comment:
The logic for verifying the update operation by comparing objects might not be reliable. Consider implementing a more robust method to verify the success of the update operation.